drivers/nrf_radio_802154/CMakeLists.txt

```diff
@@ -1,9 +1,7 @@
 zephyr_include_directories(.)
-zephyr_include_directories(rsch)
-zephyr_include_directories(rsch/raal)
+zephyr_include_directories(../../nrfx/hal) # For nrf_radio.h
```

Comment:
Well, not only for `nrf_radio.h`. Other HALs, like EGU, GPIOTE, PPI, and TIMER are also included improperly (without `hal/` before the file name) from these sources. So this is a workaround for these inclusions, which hopefully would be in some future corrected in the original driver.

Reply:
Correct, thanks for clarifying. I'll make the comment more specific. Or, if you prefer, I could just fix the inclusions as a part of the "zephyr fixes".

Reply:
Now I've noticed it was fixed in the previous version, I will just do the same now.